src/Middleware/AssignRequestId.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Honeybadger\HoneybadgerLaravel\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Str;
+use Symfony\Component\HttpFoundation\Response;
+
+class AssignRequestId
+{
+    /**
+     * Add request id to the log context.
+     *
+     * @param Closure(Request): (Response) $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if ($request->headers->has('Request-Id')) {
```

Comment:
"X-Request-Id" is still a widely used header. Can we check for both?

Reply:
Sure, done!